### PR TITLE
Test for IIIF identifiers

### PR DESCRIPTION
**Pull request by nlisgo**
This test should fail and the fix is here: https://github.com/elifesciences/journal-cms/pull/324

### CODE REVIEW

**Comment by giorgiosironi** [comment]
The failures look understandable. Probably to be merge into the other branch rather than `develop`.

File: src/modules/jcms_rest/tests/src/Unit/JCMSImageUriTraitTest.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Tests for methods in JCMSImageUriTrait.
+ *
+ * @package Drupal\Tests\jcms_rest\Unit
+ */
+class JCMSImageUriTraitTest extends UnitTestCase {
+
+  use JCMSImageUriTrait;
```
Comment (thewilkybarkid) [inline src/modules/jcms_rest/tests/src/Unit/JCMSImageUriTraitTest.php]: I'd say that mixing production code with tests shows that `JCMSImageUriTrait` needs to be refactored, but this is better than nothing.

**Comment by thewilkybarkid** [comment]
Changed the base and re-running.

**Comment by thewilkybarkid** [comment]
`travis-ci/pr` should be the only green build as it tests the PR not the branch.

**Pull request merged**